qa/L0_backend_python/lifecycle/lifecycle_test.py

```diff
@@ -58,6 +58,27 @@ class LifecycleTest(tu.TestResultCollector):
     def setUp(self):
         self._shm_leak_detector = shm_util.ShmLeakDetector()
 
+    def test_error_code(self):
+        model_name = "error_code"
+        shape = [1, 1]
+        error = "TRITONSERVER_ERROR_INVALID_ARG"
```

Comment:
Great testing addition!

I found some of this to be confusing until I reviewed the PR closer. Is there a way to simplify the model and naming? For example, the model doesn't need all the error codes if we're only going to test one, unless we expect to reuse the model for other errors soon. I think it'd be good if this can be cleaned up to make it clear that the model takes an error string input and returns a message with the associated error code followed by the error string input. That might come down to updating the variable naming and/or message.

Reply:
How about we test all the codes, and also default code on the same test? [Add all supported error codes into test](https://github.com/triton-inference-server/server/pull/6228/commits/ab83a3db5f8cbe5d73e72552fe01b271957546a7)

Reply:
Fantastic, this looks great!